wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/WorkspaceDetail.java

```diff
@@ -21,17 +21,27 @@
 
 import java.util.List;
 import java.util.Objects;
+import javax.annotation.Nullable;
 
 /**
  * Full information about the workspace, including sample data
  */
 public class WorkspaceDetail {
   private final Workspace workspace;
   private final List<Row> sample;
+  @Nullable
+  private final byte[] sampleAsBytes;
 
   public WorkspaceDetail(Workspace workspace, List<Row> sample) {
     this.workspace = workspace;
     this.sample = sample;
+    this.sampleAsBytes = null;
+  }
+
+  public WorkspaceDetail(Workspace workspace, List<Row> sample, byte[] sampleAsBytes) {
```

Comment:
mark the constructor argument as nullable.

Reply:
With latest changes this is non nullable.